src/features/quality-stats/StatsAggregator.js

```diff
@@ -57,11 +57,13 @@ class StatsAggregator {
             concealedPercentage: 0
         };
 
-        const { packets, packetsLost, samplesReceived, concealedSamples } = trackStats;
+        const { packets, packetsLost, samplesReceived, concealedSamples, startTime, endTime } = trackStats;
 
         if (!packets.length) {
             return stats;
         }
+        stats.startTime = getSQLTimestamp(startTime);
+        stats.endTime = getSQLTimestamp(endTime);
```

Comment:
I think this conversion is a level too high for my taste. I would prefer if this conversion happens in the Firehose connector. That would leave the option to write data into a non-SQL DB if needed.